app/views/application_drafts/_student_fields.html.slim

```diff
@@ -9,17 +9,15 @@
   = s.input :name, label: 'Student name', hint: 'Please provide your first and last name.', input_html: { disabled: !may_edit?(s.object) }
   .form-inline
     = s.input :application_age, :collection => ApplicationDraft::AGE, include_blank: true, label: 'Your age', disabled: !may_edit?(s.object), wrapper_html: { class: student_field_error_class(index, application_draft, :application_age)}, hint: student_field_error_hint(index, application_draft, :application_age)
-  = s.input :application_gender_identification, as: :string, label: 'Your gender self-identification', input_html: { disabled: !may_edit?(s.object), "data-genders-src" => ApplicationDraft::GENDER_LIST.to_json, class: "gender-form-field"}, wrapper_html: { class: student_field_error_class(index, application_draft, :application_gender_identification) }, hint: (student_field_error_hint(index, application_draft, :application_gender_identification) ||  'No-one is excluded from applying on the basis of gender, but people who self-identify as female, or have experiences being socialized as female, are given preference during selection.')
-  = s.input :application_diversity, as: :character_limited, label: 'Tell us about any special circumstances which you want us to consider while reviewing your application', input_html: { disabled: !may_edit?(s.object) }, wrapper_html: { class: student_field_error_class(index, application_draft, :application_diversity) }, hint: (student_field_error_hint(index, application_draft, :application_diversity) || 'E.g., LGBTQI, people of color, people with disabilities, single parents, people taking care of family members, etc. Please do not mention your name, age and location here.')
+  - unless in_same_team_as_other_student?(student)
```

Comment:
See my comment in the helper file above - I think this problem is already solved elsewhere.